hvplot/converter.py

```diff
@@ -1026,6 +1026,12 @@ def __call__(self, kind, x, y):
             elif self.datatype == 'xarray':
                 import xarray as xr
                 if isinstance(data, xr.Dataset):
+                    if kind == 'image':
```

Comment:
How about quadmeshes?

Reply:
And contours and contourf?

Reply:
Yup, all need to be handled. PR is still WIP and not ready for review.